packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormDropdown/TransferFormDropdown.tsx

```diff
@@ -52,6 +52,17 @@ const TransferFormDropdown: React.FC<TProps> = ({ fieldName, mobileAccountsListR
     const toAccountLoginId =
         location.pathname === '/wallets/cashier/transfer' ? location.state?.toAccountLoginId : undefined;
 
+    useEffect(() => {
+        // This sets a 'To transfer' to USD Wallet account to be selected by default when user transfers from a crypto wallet
+        if (!toAccount && toAccountList.walletAccounts.length) {
```

Comment:
would it make sense to put this logic into FOrmik intiialValues? 

useEffect generally tends to be really hard to maintain later on (or even figure out what it really does).
While explicitly using initialValues is obvious and very explicit for any future readers.

Reply:
In this case the initial values is dependent on the response from accounts list, thus it would need to be updated once the response is ready. But I can remove the `useEffect` and condition it instead, I agree that using more `useEffect` makes the code harder to read, will try to avoid using it in the future 😅

Reply:
Yeap agreed, I think I should avoid using useEffects from now, it makes the code harder to read indeed 😅

Reply:
ah, I see now, but if thats the case, would it make sense to wait for all the data to be loaded, before disaplying the form? 
that way, we just have all the data for the initial render,